prowler/providers/azure/lib/service/service.py

```diff
@@ -1,30 +1,30 @@
 from prowler.lib.logger import logger
-from prowler.providers.azure.lib.audit_info.models import Azure_Audit_Info
+from prowler.providers.azure.azure_provider_new import AzureProvider
 
 
 class AzureService:
     def __init__(
         self,
         service: str,
-        audit_info: Azure_Audit_Info,
+        provider: AzureProvider,
     ):
         self.clients = self.__set_clients__(
-            audit_info.identity.subscriptions,
-            audit_info.credentials,
+            provider.identity.subscriptions,
+            provider.session,
```

Comment:
The goal is to have a general name for the provider session, we were using session in AWS and credentials in Azure and GCP, but only because the internal classes for auth and session are called `Credentials`

I think session is more general and suits for all the existing and future providers